Reader.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+
+class Reader:
+
+    weathers = []
+    header_row = False
+
+    def read_file(self, name):
+        with open(name) as day:
+            if self.header_row:
+                next(day)
+            for record in day:
+                record = record.split(",")[0:10]
```

Comment:
If file format changes, we have to update this. Does it look a good and scalable approach?